content/components/selectpanel.mdx

```diff
@@ -9,51 +9,156 @@ import {Box} from '@primer/react'
 
-### Anchoring
+If the select panel doesn't have an additional checkbox in the footer, you can make an instant selection. This means that the select panel will close as soon as you make the selection.
 
-Select panel should be anchored to its trigger. 
+However, if there is an additional form element required for that selection, such as checking an additional option, it is necessary to have a cancel and confirmation button. This prevents the dialog from unexpectedly closing when you click an option, allowing you to review and confirm your selection.
 
-<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
-  <div>
-    <img
-      src="https://user-images.githubusercontent.com/40274682/229016799-9559a4ee-0daf-42d1-b46d-11c64cf182e4.png"
-      alt=""
-    />
-    <Caption>Anchored select panel to a trigger.</Caption>
-  </div>
-</Box>
+To indicate an instant selection, we use checkmarks to show the selected option. For regular single selects, we use visually styled radio buttons. Note they are not semantically radio buttons.
+
+### List (multi select)
+
+<img
+  width="960"
+  alt="Multi-select"
+  src="https://github.com/primer/design/assets/980622/33176d07-2853-4318-bf5d-df395e700fa9"
+/>
+
+The multi-select list allows you to choose multiple items from a list and has additional features like `select all` or `indeterminate selections`.
+
+The `select all` option at the top of the list allows for quickly deselecting or selecting all values
+however when you seperate items in groups this option isn't available anymore.
+
+It also can only be used when all available options can be returned. For example if the list is too large to retrieve all options at once,
+such as when using API endpoints with limited results then this feature may not be suitable.
+We should refrain from putting a strain on API endpoints to return all the possible results at once as this could result in a performance downgrade in a scenario of large lists since virtualization is currently not built into this component.
+
+The `intermediate selections` option is useful for performing bulk actions on lists. For example, if you have 20 selected items, but some of them have different labels attached, an indeterminate checkbox (represented by a horizontal line) is shown for labels that are not set on all items.
+
+### Footer
+
+<img
+  width="1388"
+  alt="Footers"
+  src="https://github.com/primer/design/assets/980622/85bcb93b-3d39-4635-ad1b-68243dae6bd9"
+/>
+
+The optional footer at the bottom could include a single link, button or
+checkbox to take additional actions.
+
+In case of a multi selection or a single selection with additional actions in the footer we always recommend a `cancel` and `apply` button unless you clearly [understand the accessibility implications](#cancelsubmit) of not adding this.
```

Comment:
```suggestion
In case of a multi selection or a single selection with additional actions in the footer, we always recommend a `cancel` and `apply` button unless you clearly [understand the accessibility implications](#cancelsubmit) of not adding this.
```